solutions/sumAnArray/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Title</title>
+</head>
+<styles></styles>
+
+<body>
+
+  <script>
+function sumArray(array){
+  var total=0;
+  for(var i in array){
```

Comment:
I would not recommend getting into the habit of using for in loops for arrays. When you are doing a full loop over the contents of an array, the best path to follow is a for loop. Even though it works, for in loops give people the impression they are working with an object, which can be confusing. Additionally, you have no control over the loop itself, so you can't move backwards or forwards in your array by adjusting your incrementer value, if you ever had the need.